stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/home/HomeFragment.kt

```diff
@@ -80,6 +81,8 @@ class HomeFragment : Fragment() {
 
                 requireActivity().intent = null
 
```

Comment:
This is a hack. We did everything in push notification handler except connecting to the socket.